packages/material-ui-based-components/src/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useCallback } from 'react';
+import { withRuleEngine } from '../shared/withRuleEngine';
+import { PROPS } from '../utils/types';
+import Button from '@mui/material/Button';
+
+const ButtonComponent = (props: PROPS) => {
+
+  const { label, enabled, id, name } = props;
+
+  const clickHandler = useCallback((event: any) => {
+    props.dispatchClick(event);
+  }, [props.dispatchClick]);
+
+  return (
+    <Button variant="outlined" disabled={!enabled} onClick={clickHandler} id={id} name={name}>
```

Comment:
variant should be dynamic. It will come from the props.properties['afs:layout'].variant and default value of this outlined

Reply:
Added variant